policies/clair/vulnerabilities-check.rego

```diff
@@ -1,42 +1,45 @@
 package required_checks
 
-violation_critical_vulnerabilities[{"msg": msg, "details":{"name": name, "description": description, "url": url}}] {
+violation_critical_vulnerabilities[{"msg": msg, "vulnerabilities_number": vulns_num, "details":{"name": name, "description": description, "url": url}}] {
   rpms_with_critical_vulnerabilities := {rpm.Name | rpm := input.data[_].Features[_]; count(rpm.Vulnerabilities) > 0; rpm.Vulnerabilities[_].Severity == "Critical"}
   not count(rpms_with_critical_vulnerabilities) == 0
 
   name := "clair_critical_vulnerabilities"
-  msg = sprintf("Found packages with critical vulnerabilities: %s", [concat(", ", rpms_with_critical_vulnerabilities)])
-  description := "The image musn't contain packages that have critical severity vulnerabilities."
-  url := "https://source.redhat.com/groups/public/product-security/content/product_security_folder/understanding_red_hat_security_vulnerabilities_10pdf"
+  vulns_num = count(rpms_with_critical_vulnerabilities)
+  msg := "Found packages with critical vulnerabilities. Consider updating to a newer version of those packages, they may no longer be affected by the reported CVEs. You can check clair logs for more infromation."
```

Comment:
There's a typo on "infromation", but personally, I don't see a lot of value in saying "You can check clair logs for more infromation." with each category message. They're viewing that log right now, they just need to scroll up to the previous step.

Reply:
Everything else looks good to me @jsztuka, thanks for making the other changes!

Reply:
Included the update note change in previously mentioned story.